_pages/en/strategy-communications.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Communications Strategy
+ref: strategy-communications
+lang: en
+status: posted
+sections: Work In Progress
+permalink: /strategy-communications.html
+---
+
+## Communications Strategy
+
+- [Endorsement](#Endorsement)
```

Comment:
Add Heading "Evaluation" or "Key Performance Indicators"

How will we measure ourselves on whether our communication strategy works or needs to be adjusted?
KPIs should be closely related to objectives but can also be used to support results in our actions. E.g.

- Number of hits on our website
- Number of endorsements received
- Number of forum events the team presented at
- Number of attendance at those forum events
- Number of feedbacks received as part of the Feedback loop Communications method
- Number of twitter retweets from CIO/IITB stakeholders